src/Main.hs

```diff
@@ -90,9 +90,24 @@ getExecutables = do
         [] -> error "No executables found"
         ds -> map fst ds
 
+getCabalProjectRootCurrent :: IO (Maybe FilePath)
+getCabalProjectRootCurrent = flip catchIOError (const (return Nothing)) $ do
```

Comment:
I'm not sure if `flip catchError (const (return Nothing))` is necessary here, but I thought I'd leave it in just to be sure.